src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <script
+      type="text/javascript"
+      src="./main.js">
+    </script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="formfild">
+        <legend>Personal information</legend>
+        <div class="form">
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="sname"
```

Comment:
And please provide more logically correct name, for example surname in this case. Please fix for all inputs!